modules/PyEvJosev/utilities.py

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+import logging
+import netifaces
+
+from everest.framework import log
+
+from iso15118.evcc.evcc_config import EVCCConfig
+from iso15118.shared.utils import load_requested_protocols
+
+class EverestPyLoggingHandler(logging.Handler):
+
+    def __init__(self):
+        logging.Handler.__init__(self)
+
+    def emit(self, record):
+        msg = self.format(record)
+
+        log_level: int = record.levelno
+        if log_level == logging.CRITICAL:
+            log.critical(msg)
+        elif log_level == logging.ERROR:
+            log.error(msg)
+        elif log_level == logging.WARNING:
```

Comment:
This is, I guess, exactly the same as in the EVSE part.  I'd prefer removing this duplication by creating a shared library for this inside 'everest-core/lib' - again we would need to think on how to package and distribute/install that later on

Reply:
It's the same code